js/view/printView.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * @param {jQuery object} container - references the HTML parent element that contains the view.
+ * @param {Object} model - the reference to the Dinner Model
+ */
+
+var PrintView = function (container, model) {
+    var numberOfGuests = container.find(".numberOfGuests");
```

Comment:
In the rest of our JS code we have used two-space indentation; could you maybe also do that here for consistency?